### ISSUE TITLE
iOS 12.4.1 support for 6s plus

### ISSUE THREAD

**Issue by eglacias** [pwn20wndstuff/Undecimus#1923]
Can we get iOS 12.4.1 support for iPhone 6S plus (a9)? I get unsupported message as it is not supported I have confirmed.

I’m definitely willing to beta-test any solution. I have a number of older tweaks and even apps that will still not work on iOS 13.

The Only other thread that was relevant is now closed. In that thread other people also had issues with the three pieces of hardware that do support 12.4.1 with uncover.

**Comment by asdfugil** [comment]
well it will happen sometime soon™ because time_waste works on ios 12.

**Comment by theLaxerz** [comment]
Why not just use Checkra1n if Unc0ver has not incorporated or does Checkra1n not have one either being if that’s the case is it because no exploit was ever found to bypass the Apple security model and allow root. 

Sincerely,
Aaron Lax

**Comment by eglacias** [comment]
We do use checkrain on it. Works ok. What we want is an *on device jailbreak*. Having to find a computer anytime I lose power is not convenient. Reprovision  also works to sign uncover right now.  There it sits signed, saying “unsupported“

**Comment by eglacias** [comment]
Is there any chance of ever getting support for this version? It would be nice if it’s not going to happen or for some seemingly impossible reason can’t happen, for:
One of the developers to just mark this not gonna happen so I don’t have to Check up on him covered anymore for this phone!
 I would be equally happy happy to see that going to happen :-)

**Comment by asdfugil** [comment]
I think it's now supported.

**Comment by eglacias** [comment]
Yes it’s technically supportive but not working. If I restore rootfs with checkrain then uncover after reboot, then uncover jailbreak install cydia filza and use batchomatic to reinstall all of my tweaks, everything runs perfectly and I can respring too. Now if I reboot, it gets stuck on the white Apple. 

At that point even though it jailbroke it self and I could install all of my tweaks and run them, it failed to jailbreak again on a reboot. If I unload tweaks, it will jailbreak which of course is useless.

 Regular iOS 12.4 worked perfectly fine with the old uncover with the exact same tweaks 

 I don’t have tweaks that I don’t need, so I’ll go through the painful process of removing stuff until it finally jailbreaks and we’ll see what happens. No fun at all. 

Oh I forgot to mention, jailbreaking with check rain after a reboot, all of my tweaks are still unload. That’s too bad because I was going to save my current tweak state before I start removing stuff. Now I’m stuck with no ability to do so

**Comment by eglacias** [comment]
Watchdog pro seem to be the culprit. That’s really too bad because there is no backgrounding tweak at all other than that that actually backgrounds apps. it was working fine with old uncover and checkrain. It also works if I install it and only do a respring.

**Comment by eglacias** [comment]
Checkrain is not working with version five. I’m guessing substitute or Cydia substrate is not the same Anymore or something? 
 It gets the jailbreak, but has me in safe mode essentially without telling me it is in safe mode

**Comment by eglacias** [comment]
I guess I still won’t close this thread because:
Checkrain no longer works with uncover.

 Uncover it self is working marvelously however. I noticed the Reddit page on what is jailbreakable has not been updated either. 

Why is substitute and substrate both used interchangeably but mutually exclusive between these two and the 2 jailbreaks?

Basically checkrain is in safe mode and will not leave safe mode.

**Comment by eglacias** [comment]
Another reason not to close this thread is because I enumerated a very important tweak which was functioning perfectly well using substrate instead of substitute on uncover 5: watchdog pro.  It may be a very old application, but it was still working perfectly right up to iOS 12.4.1 which I run,  and there is no other background application in the world other than this one

**Comment by SarKaa** [comment]
What don't you people understand, ios 12.4.1 to 13 (not inclusive) is not bloody compatible. I know unc0ver says all versions are, but you should read around, read the fine print. You should appreciate what goes into a jailbreak, and you can't expect everything you want to just happen. It is currently impossible to jailbreak you device, and there is no way you can change that, unless you update to ios 13.5. 
Edit: sorry this was posted on the wrong thread, I didn't mean to comment here, sorry

**Comment by eglacias** [comment]
I am running iOS 12.4.1. With uncover 5. On an iPhone 6s Plus. It is compatible. 
On It is entirely compatible. I have one single tweak that doesn’t work and everything else is working more stable than any jailbreak I have ever run.

**Comment by SarKaa** [comment]
> I am running iOS 12.4.1. With uncover 5. On an iPhone 6s Plus. It is compatible.
> On It is entirely compatible. I have one single tweak that doesn’t work and everything else is working more stable than any jailbreak I have ever run.

Oh sorry I put my comment on the wrong thread, and I meant anything in between 12.4.1 and 13, not inclusive, sorry for the confusion

**Comment by rlatm** [comment]
@eglacias - Based on the iOS jailbreak chart ( https://docs.google.com/spreadsheets/d/11DABHIIqwYQKj1L83AK9ywk_hYMjEkcaxpIg6phbTf0/ ), Chimera ( https://chimera.coolstar.org ) is also listed as supporting your device and I didn't see it mentioned so far here. Could it work for you?
